platform/ext/target/nordic_nrf/nrf9160/spm_hal.c

```diff
@@ -0,0 +1,323 @@
+/*
```

Comment:
What about making this file a common one for nRF5340 and nRF9160? Currently, for both platforms it is identical.

Reply:
I think it's good idea.

Reply:
I agree, as long as we are confident we are not going to have #ifdef PLATFORM directives there.

Reply:
Moved the file to `nordic_nrf/common/`.